nixos/modules/services/networking/sickbeard.nix

```diff
@@ -0,0 +1,79 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.sickbeard;
+  sickbeard = cfg.package;
+in
+{
+
+  ###### interface
+
+  options = {
+    services.sickbeard = {
+      enable = mkOption {
+        default = false;
+        description = "Whether to enable the sickbeard server.";
+      };
+      package = mkOption {
+        type = types.package;
+        default = pkgs.sickbeard;
+        example = literalExample "pkgs.sickrage";
+        description =''
+          Enable <literal>pkgs.sickrage</literal> or <literal>pkgs.sickgear</literal>
+          as an alternative to SickBeard
+        '';
+      };
+      dataDir = mkOption {
+        default = "/var/lib/sickbeard";
+        description = "Path where to store data files.";
+      };
+      configFile = mkOption {
+        default = "/var/lib/sickbeard/config.ini";
+        description = "Path to config file.";
+      };
+      port = mkOption {
+        default = 8081;
+        description = "Port to bind to.";
+      };
+      user = mkOption {
+        default = "sickbeard";
+        description = "User to run the service as";
+      };
+      group = mkOption {
+        default = "sickbeard";
+        description = "Group to run the service as";
+      };
+    };
+  };
+
+
+  ###### implementation
+
+  config = mkIf cfg.enable {
+
+    users.users.sickbeard = {
+          uid = config.ids.uids.sickbeard;
+          group = "sickbeard";
+          description = "sickbeard user";
+          home = "/var/lib/sickbeard/";
+          createHome = true;
+    };
+
+    users.groups.sickbeard = {
+      gid = config.ids.gids.sickbeard;
+    };
+
+    systemd.services.sickbeard = {
+        description = "sickbeard server";
```

Comment:
According to #45638:

"I don't remember where we have that documented, but service descriptions should be upper-case-first for every word."